libs/server-sdk/README.md

```diff
@@ -0,0 +1,121 @@
+
+Getting started
+---------------
+
+Download a release archive from
+the [Github releases](https://github.com/launchdarkly/cpp-sdks/releases?q=cpp-server&expanded=true) for use in your
+project.
+
+Refer to the [SDK documentation][reference-guide] for complete instructions on
+installing and using the SDK.
+
+### Incorporating the SDK
+
+The SDK can be used via a C++ or C interface and can be incorporated via a static library or shared object. The static
+library and shared object each have their on use cases and limitations.
+
+The static library supports both the C++ and C interface. When using the static library, you should ensure that it is
+compiled using a compatible configuration and toolchain. For instance, when using MSVC, it needs to be using the same
+runtime library.
+
+Using the static library also requires that you have OpenSSL and Boost available at the time of compilation for your
+project.
+
+The C++ API does not have a stable ABI, so if this is important to you, consider using the shared object with the C API.
+
+Example of basic compilation using the C++ API with a static library using gcc:
+
+```shell
+g++ -I path_to_the_sdk_install/include -O3 -std=c++17 -Llib -fPIE -g main.cpp path_to_the_sdk_install/lib/liblaunchdarkly-cpp-server.a -lpthread -lstdc++ -lcrypto -lssl -lboost_json -lboost_url
+```
+
+Example of basic compilation using the C API with a static library using msvc:
+
+```shell
+cl /I include /Fe: hello.exe main.cpp /link lib/launchdarkly-cpp-server.lib
+```
+
+The shared library (so, DLL, dylib), only supports the C interface. The shared object does not require you to have Boost
+or OpenSSL available when linking the shared object to your project.
+
+Example of basic compilation using the C API with a shared library using gcc:
+
+```shell
+gcc -I $(pwd)/include -Llib -fPIE -g main.c liblaunchdarkly-cpp-server.so
+```
+
+The examples here are to help with getting started, but generally speaking the SDK should be incorporated using your
+build system (CMake for instance).
+
+Learn more
+-----------
+
+Read our [documentation](https://docs.launchdarkly.com) for in-depth instructions on configuring and using LaunchDarkly.
+You can also head straight to
+the [complete reference guide for this SDK][reference-guide].
+
+Testing
+-------
+
+We run integration tests for all our SDKs using a centralized test harness. This approach gives us the ability to test
+for consistency across SDKs, as well as test networking behavior in a long-running application. These tests cover each
+method in the SDK, and verify that event sending, flag evaluation, stream reconnection, and other aspects of the SDK all
+behave correctly.
+
+Contributing
+------------
+
+We encourage pull requests and other contributions from the community. Check out
+our [contributing guidelines](../../CONTRIBUTING.md) for instructions on how to contribute to this SDK.
+
+About LaunchDarkly
```

Comment:
(This section also has some minor changes in the `sdk-docs` repo, but we haven't rolled them out anywhere, so I think it's fine to leave this as-is.)